get/README.md

```diff
@@ -6,4 +6,11 @@ The easiest way to get HACS for Home Assistant.
 
 1. Use the **Install** button above to build the add-on.
 2. Start the add-on.
-3. Navigate to the add-on logs and follow instructions given there.
+3. Navigate to the add-on logs and follow the instructions given there.
+
+> [!NOTE]
```

Comment:
The supervisor frontend does not currently support this format.

Reply:
We could also do an API call to restart core automatically?

Reply:
I thought about it, buttoo much magic will be confusing i think,
Maybe later as an optional config option